Description: Upstream changes introduced in version 2010.09-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 u-boot (2010.09-1) unstable; urgency=low
 .
   * New upstream release.
 .
 The person named in the Author field signed this changelog entry.
Author: Clint Adams <schizo@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- u-boot-2010.09.orig/snapshot.commit
+++ u-boot-2010.09/snapshot.commit
@@ -1 +1 @@
-1e4e5ef0469050f014aee1204dae8a9ab6053e49  Tue, 28 Sep 2010 23:20:55 +0200
+$Format:%H  %cD$
